update-dependencies/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.VersionTools;
+using Microsoft.DotNet.VersionTools.Automation;
+using Microsoft.DotNet.VersionTools.Dependencies;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Dotnet.Docker.Nightly
+{
+    public static class Program
+    {
+        private static readonly string s_repoRoot = Directory.GetCurrentDirectory();
+        private static readonly Config s_config = Config.s_Instance;
+        private static bool s_updateOnly = false;
+
+        public static void Main(string[] args)
+        {
+            if (ParseArgs(args))
+            {
+                DependencyUpdateResults updateResults = UpdateFiles();
+
+                if (!s_updateOnly && updateResults.ChangesDetected())
+                {
+                    CreatePullRequest(updateResults).Wait();
+                }
+            }
+        }
+        
+        private static bool ParseArgs(string[] args)
+        {
+            foreach (string arg in args)
+            {
+                if (string.Equals(arg, "--Update", StringComparison.OrdinalIgnoreCase))
+                {
+                    s_updateOnly = true;
+                }
+                else if (arg.Contains('='))
+                {
+                    int delimiterIndex = arg.IndexOf('=');
+                    string name = arg.Substring(0, delimiterIndex);
+                    string value = arg.Substring(delimiterIndex + 1);
+
+                    Environment.SetEnvironmentVariable(name, value);
+                }
+                else
+                {
+                    Console.Error.WriteLine($"Unrecognized argument '{arg}'");
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
+        private static DependencyUpdateResults UpdateFiles()
+        {
+            IEnumerable<BuildInfo> buildInfos = new[] { BuildInfo.Get("Cli", s_config.CliVersionUrl, fetchLatestReleaseFile: false) };
+            IEnumerable<IDependencyUpdater> updaters = GetUpdaters();
+
+            DependencyUpdater updater = new DependencyUpdater();
+            return updater.Update(updaters, buildInfos);
+        }
+
+        private static Task CreatePullRequest(DependencyUpdateResults updateResults)
+        {
+            string commitMessage = $"Update CLI to {updateResults.UsedBuildInfos.Single().LatestReleaseVersion}";
```

Comment:
>CLI [](start = 44, length = 3)

What do you think about using the term SDK instead of CLI?  In general the Dockerfiles use the SDK nomenclature.